lochness/utils/source_check.py

```diff
@@ -311,21 +311,28 @@ def check_source(Lochness: 'lochness', test: bool = False) -> None:
 
     # select final_check failed files, and clean up
     qc_fail_df = all_df[
-            (~all_df['final_check']) | (~all_df['exist_in_db'])
+            (~all_df['final_check']) | 
+            (~all_df['exist_in_db']) |
+            (~all_df['subject_check'])
             # (~all_df['file_name'].str.contains('.dcm|._dicom_series'))
             ]
 
+    qc_fail_df['subject_check'] = qc_fail_df['subject_check'].map(
+            {True: f'Correct', False: f'Incorrect'})
```

Comment:
Wouldn't it have been easier to code `Correct` as 1 and `Incorrect` as 0?

Reply:
No, this email is for non-engineers. I think it would be better to keep it as a word.